JBBCode/Parser.php

```diff
@@ -97,7 +97,7 @@ public function addBBCode($tagName, $replace, $useOption = false, $parseContent
      */
     public function addCodeDefinition(CodeDefinition $definition)
     {
-        $this->bbcodes[] = $definition;
+        $this->bbcodes[$definition->getTagName()][$definition->usesOption()] = $definition;
```

Comment:
Actually yes. But PHPUnit doesn't complain with `convertWarningsToExceptions` and/or `convertNoticesToExceptions` …